src/app/backend/resource/job/detail.go

```diff
@@ -95,3 +99,50 @@ func getJobDetail(job *batch.Job, metricClient metricapi.MetricClient,
 		Completions:     job.Spec.Completions,
 	}
 }
+
+// DeleteJob with given name in given namespace and related pods.
+func DeleteJob(client k8sClient.Interface, namespace, name string) error {
+
+	log.Printf("Deleting %s job from %s namespace", name, namespace)
+
+	job, err := client.BatchV1().Jobs(namespace).Get(name, metaV1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	count := 0
+	*job.Spec.Parallelism = int32(count)
+	_, err = client.BatchV1().Jobs(namespace).Update(job)
```

Comment:
Thanks， I did not think so well.  I will try and learn to write this code again.